test/fw/ptl/utils/plugins/ptl_test_db.py

```diff
@@ -1615,16 +1615,19 @@ def __init__(self, dbtype, dbpath, dbaccess):
         self.res_data = PTLJsonData(command=self.__cmd)
 
     def __write_test_data(self, data):
-        jdata = None
         if _TESTRESULT_TN not in self.__dbobj.keys():
-            self.__dbobj[_TESTRESULT_TN] = open(self.dbpath, 'w+')
+            self.__dbobj[_TESTRESULT_TN] = self.dbpath
```

Comment:
isn't self.__dbobj[_TESTRESULT_TN] redundant now? you already have the instance variable self.dbpath, and it seems like self.__dbobj[_TESTRESULT_TN] only gets set to one value: self.dbpath. Why not just do open("self.dbpath") instead in this class?

Reply:
Yes, it is redundant now.  I will remove self.__dbobj.